impeller/renderer/backend/metal/surface_mtl.mm

```diff
@@ -16,10 +17,10 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunguarded-availability-new"
 
-std::unique_ptr<SurfaceMTL> SurfaceMTL::WrapCurrentMetalLayerDrawable(
+id<CAMetalDrawable> SurfaceMTL::GetMetalDrawableAndValidate(
     const std::shared_ptr<Context>& context,
     CAMetalLayer* layer) {
-  TRACE_EVENT0("impeller", "SurfaceMTL::WrapCurrentMetalLayerDrawable");
+  TRACE_EVENT0("impeller", "SurfaceMTL::GetMetalDrawableAndValidate");
```

Comment:
Are both of these events (this and the one int he new method) helpful?

Reply:
Probably not!, I removed one